index.bs

```diff
@@ -178,7 +178,7 @@ interface SpeechRecognition : EventTarget {
     attribute boolean interimResults;
     attribute unsigned long maxAlternatives;
     attribute boolean processLocally;
-    attribute SpeechRecognitionPhraseList phrases;
+    attribute sequence<SpeechRecognitionPhrase> phrases;
```

Comment:
This is invalid Web IDL; attributes cannot be sequences.

ObservableArray is probably the correct choice here. See https://webidl.spec.whatwg.org/#idl-observable-array